e2e/e2e.go

```diff
@@ -76,18 +75,26 @@ func BuildDependenciesList(ctx context.Context) ([]string, error) {
 
 // findDependenciesInRepo finds the dependencies in a repository
 // at all of the possible paths
-func findDependenciesInRepo(serviceName, repoPath string) map[string]struct{} {
+func findDependenciesInRepo(ctx context.Context, serviceName string) (map[string]struct{}, error) {
 	possibleFiles := []string{"devenv.yaml", "noncompat-service.yaml", "service.yaml"}
+	gh, err := githubauth.NewClient()
+	if err != nil {
+		return nil, err
+	}
 
 	var dc *DevenvConfig
 	for _, f := range possibleFiles {
-		if _, err := os.Stat(filepath.Join(repoPath, f)); err != nil {
+		config, _, _, err := gh.Repositories.GetContents(ctx, "getoutreach", serviceName, f, nil)
+		if err != nil {
 			continue
 		}
-
-		var err error
-		dc, err = parseDevenvConfig(filepath.Join(repoPath, f))
+		content, err := config.GetContent()
 		if err != nil {
+			log.Warn().Str("service", serviceName).Msgf("Unable to get content of file %s", f)
```

Comment:
Should we always warn here? I think it's expected that some will fail (e.g., `flagship`) so that seems like extra clutter. Not super attached to this, but I like to log less if it makes sense

Reply:
yeah, I wanted to keep it there only because it always was there, but I hate it TBH

Reply:
Maybe we can just throw it being `log.Debug()` for now and call it good? We should have a `--debug` flag that works

Reply:
👍  Added